doc/using_azure-iot-sdk-c_with_the_latest_tls_version_only.md

```diff
@@ -0,0 +1,52 @@
+# How to configure the Azure IoT C SDK and currently supported TLS platforms to use TLS 1.2 only 
+
+
+## SChannel (Microsoft Windows)
+
+To use exclusively TLS 1.2 in Microsoft Windows using SChannel, disable the support for TLS 1.0 and 1.1.
```

Comment:
Note: Win7/2k8 does not have support for TLS1.2 by default: a KB must be installed. Win8+ do have it enabled by default.

Reply:
Add this link to this section? 
It's a bigger deep dive into windows OS TLS configuration and OS support

https://docs.microsoft.com/en-us/dotnet/framework/network-programming/tls

Reply:
@ericwol-msft , the documentation has been added.